4.analyze-components/scripts/util.py

```diff
@@ -119,14 +141,16 @@ def get_svcca_across_z_stability(z_dict_a, z_dict_b, train_or_test='train'):
     z_dict_a - a dict of the output of `read_in_z` for specific z and dataset
     z_dict_b - the same structure but a different z and dataset than z_dict_a
     train_or_test - a string that indicates if the comparison should be
-                    training or testing
+                    training or testing (or None if weight matrix)
+    threshold - remove trailing zeros if applicable (for cca_similarity)
 
```

Comment:
What is the difference between training, testing and using the weight matrix?

Reply:
there are two z matrices (one for training and one for testing), but there is only one weight matrix. The one derived from the training data.